### ISSUE TITLE
Derack old machines in UCL

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/operations#411]
Because it's not a remote location we don't need to remotely wipe the drives.

I don't have a list of all the old machines, so this list needs expanding

- [ ] yevaud

**Comment by Firefishy** [comment]
There are at least 3 or 4 machines. Old old DB, G5 and some one-off kit.

**Comment by Firefishy** [comment]
4x 1U servers extracted from UCL. We'll need to arrange disposal for them.

**Comment by Firefishy** [comment]
Once all has been collected I will arrange WEEE recycling for the machines.

**Comment by Firefishy** [comment]
![IMG_20200614_221509](https://user-images.githubusercontent.com/248188/84604453-1179b780-ae8e-11ea-9b8b-ee4c922a867c.jpg)
Disks going for disposal. All wiped.

**Comment by Firefishy** [comment]
Blocked by https://github.com/openstreetmap/operations/issues/1060

**Comment by Firefishy** [comment]
Closing in favour of: https://github.com/openstreetmap/operations/issues/1088
